src/kem/kyber/ind_cpa.rs

```diff
@@ -207,8 +205,8 @@ pub(crate) fn encrypt<
     const T_AS_NTT_ENCODED_SIZE: usize,
     const C1_LEN: usize,
     const C2_LEN: usize,
-    const VECTOR_U_COMPRESSION_FACTOR: usize,
-    const VECTOR_V_COMPRESSION_FACTOR: usize,
+    const U_COMPRESSION_FACTOR: usize,
```

Comment:
For future, please split out changes like this into a separate PR.

Reply:
Understood, I got a little carried away with this PR ...